src/rfm.rs

```diff
@@ -353,6 +376,46 @@ where
         self.write(Registers::TestLna, boost as u8)
     }
 
+    pub fn tx_level(&mut self, power_level: i8) -> Result<(), Espi> {
```

Comment:
The function is missing documentation comment that would describe the limitation of `power_level`, also `tx_level` isn't very descriptive name, I would suggest `tx_power` instead WDYT?